<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/images/bg.png);
border: 0px </string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="CAR0">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>590</y>
      <width>13</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color : none;
color : blue;</string>
    </property>
    <property name="text">
     <string>CAR0</string>
    </property>
   </widget>
   <widget class="QLabel" name="CAR1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>590</y>
      <width>13</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color : none;
color : red;</string>
    </property>
    <property name="text">
     <string>CAR1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>119</width>
      <height>62</height>
     </rect>
    </property>
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionSTART"/>
   </widget>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSTART">
   <property name="text">
    <string>START</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
